server/service/datadeal/service_data_validation_test.go

```diff
@@ -318,3 +318,64 @@ func (suite *dataDealServiceServerTestSuite) TestValidateDataInvalidJSONSchema()
 	suite.Require().Nil(res)
 	suite.Require().ErrorContains(err, "failed to validate data")
 }
+
+func (suite *dataDealServiceServerTestSuite) TestCanonicalJSON() {
```

Comment:
The test is good, but it would be better to create a new test like `json_canonical_test.go` rather than `service_data_validation_test.go`.
I think it's a test that doesn't belong here.

Reply:
I agree it. Fixed it in [7ecfa10](https://github.com/medibloc/panacea-oracle/pull/117/commits/7ecfa10e2a960921d6aff707af4fba2457c745f7).